Dockerfile

```diff
@@ -1,23 +1,9 @@
-FROM python:3.8.5-slim-buster
-
-ARG TOKEN=none
-ARG WEBHOOK_ENABLED=false
-ARG WEB_URL=none
-ARG GROUPS=none
-
-ENV BOT_DIR    /app
-
-WORKDIR $BOT_DIR
+FROM python:3
```

Comment:
Can you use python:3-slim?

Reply:
Nope, some libraries required some packages that are not included in python:3-slim